django/db/models/expressions.py

```diff
@@ -536,6 +539,61 @@ def _prepare(self, output_field=None):
         return self
 
 
+class SlicableF(F):
```

Comment:
spelling: `Sliceable`

Reply:
How about `SlicedF`? It's not 'sliceable' at this point, it has actually been sliced.